pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -2234,6 +2234,14 @@ in
     nvimRequireCheck = "refactoring";
   };
 
+  remote-nvim-nvim = super.remote-nvim-nvim.overrideAttrs {
+    dependencies = with self; [
+      nui-nvim
+      plenary-nvim
+    ];
+    nvimSkipModule = "repro";
```

Comment:
It shouldn't be repro...

https://github.com/amitds1997/remote-nvim.nvim/blob/main/repro.lua

Reply:
Not sure what you mean? 
<img width="754" alt="image" src="https://github.com/user-attachments/assets/cb3e7b30-2910-42b1-8b35-37e37e1e99a5">

Reply:
Oh, sorry, confused this with `nvimRequireCheck`